pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -445,6 +446,88 @@ var _ = Describe("Evacuation", func() {
 
 			controller.Execute()
 		})
+
+		It("Should not evict new VMIs when the amount of concurrent running migrations reached its maximum capacity", func() {
+			var maxParallelMigrationsPerCluster uint32 = 5
+			config, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{
+				MigrationConfiguration: &v1.MigrationConfiguration{
+					ParallelMigrationsPerCluster: &maxParallelMigrationsPerCluster,
```

Comment:
It's not clear if the controller will hit the global or per-node limit. At least to me. Also, it seems too similar to "`should not evict VMIs if 5 migrations are in progress`"

Reply:
@xpivarc Correct! thanks for mention this